src/api/__main__.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+"""Code to run if this package is used as a Python module."""
+
+# Third-Party Libraries
+import connexion
+
+# cisagov Libraries
+from api import encoder
+
+
+def main():
+    """Run Main Application.."""
+    app = connexion.App(__name__, specification_dir="./openapi/")
```

Comment:
Can you check if this works?  I think it should:
```suggestion
    app = connexion.App(__name__, specification_dir="openapi")
```